xbmc/music/MusicDatabase.cpp

```diff
@@ -4104,7 +4113,7 @@ bool CMusicDatabase::GetSongsFullByWhere(const std::string &baseDir, const Filte
     {
       unsigned int targetRow = (unsigned int)it->at(FieldRow).asInteger();
       const dbiplus::sql_record* const record = data.at(targetRow);
-      
```

Comment:
Please keep all these cosmetic changes in separate commits as they have nothing to do with the actual fix.